package.json

```diff
@@ -8,13 +8,15 @@
     "babel": "^6.23.0",
     "concurrently": "^7.0.0",
     "cors": "^2.8.5",
+    "dom": "^0.0.3",
```

Comment:
And this?

Reply:
Also wondering about this

Reply:
Is this not when we installed react router dom?